doc/latex.rst

```diff
@@ -354,9 +354,9 @@ The available styling options
                                ``attentionBgColor``, ``dangerBgColor``,
                                ``errorBgColor``
 
-.. |warningborders| replace:: ``warningBorder``, ``cautionBorder``,
-                              ``attentionBorder``, ``dangerBorder``,
-                              ``errorBorder``
+.. |warningborders| replace:: ``warningborder``, ``cautionborder``,
+                              ``attentionborder``, ``dangerborder``,
+                              ``errorborder``
```

Comment:
thanks @mgeier for catching this. I guess I intended to use the camelcase names in `sphinx.sty` but indeed (in `sphinxadmonition` environment definition, the `\dimexpr` line) it is done with lowercase names... too late to change that.